src/main/java/frc/robot/subsystems/VisionControl.java

```diff
@@ -272,17 +265,48 @@ private double calculateBallRotation() {
         return -ball_rotation;
     }
 
-    public double calculateShooterRPMS() {
+    public void autoShoot() {
+        switch (shooterstate) {
+            case ALIGN:
+                trackHoop();
+                if (Math.abs(hoopr) <= hoopChassisThreshold) {
+                    shooterstate = shooterState.WAIT;
+                }
+                break;
+            case WAIT:
+                double rpm = calculateShooterRPMS();
+                shooter.startShoter(rpm);
+                if (Math.abs(shooter.getShooterRpms() - rpm) < shooterThreshold) {
+                    shooterstate = shooterState.SHOOT;
+                }
+                break;
+            case SHOOT:
+                double rpms = calculateShooterRPMS();
+                shooter.startShoter(rpms);
```

Comment:
Can just set the feed motor power for a time then be done

Reply:
done